sdks/python/apache_beam/yaml/yaml_udf_test.py

```diff
@@ -60,16 +66,37 @@ def test_map_to_fields_filter_inline_js(self):
         language: javascript
         fields:
           label:
-            callable: "function label_map(x) {return x.label + 'x'}"
+            callable: |
+              function label_map(x) {
```

Comment:
This is testing callable specifically. I wanted to make tests that only use 1 of the 3 types (expression, callable, file) so that the failures are granular.